src/__tests__/interactions-helper.js

```diff
@@ -108,6 +108,11 @@ export const waitForElementVisible = getterFunc => {
   });
 };
 
+export const keyboardMultipleTimes = async (text, times, options = { delay: 70 }) => {
```

Comment:
is the word time is the correct word? it's not about time it's about amount (can be confiusing)

Reply:
will update to "count" 👍